port/unix/omrsysinfo.c

```diff
@@ -4538,7 +4557,6 @@ omrsysinfo_cgroup_subsystem_iterator_next(struct OMRPortLibrary *portLibrary, st
 		 * which is not required so we could remove it 
 		 */
 		size_t len = strlen(metricElement->value);
-		metricElement->key = currentElement->metricTag;
 		metricElement->units = currentElement->metricUnit;
 		if ((len > 0) && (metricElement->value[len-1] == '\n')) {
 			metricElement->value[--len] = '\0';
```

Comment:
`--len` use `len-1` just like the code above as `len` is never used again.

Reply:
changed `--len` to `len-1`